### ISSUE TITLE
Disclose default metrics queries

### ISSUE THREAD

**Issue by petef19** [prometheus-community/postgres_exporter#584]
## Proposal
**Use case. Why is this important?**

For folks that want to custom query only a small subset of the default metrics queries, to minimize db traffic on each Prometheus scrape (trimming it on the Prometheus side will still unnecessarily hammer the db).

I've looked through the code and found some queries in [queries.go](https://github.com/prometheus-community/postgres_exporter/blob/master/cmd/postgres_exporter/queries.go), but cannot find the queries that get the data for the `pg_stat_*`  default metrics.

Any help appreciated.

Thanks !

**Comment by SuperQ** [comment]
Yes, we need to better document the collectors like we do in the mysqld_exporter.
